src/jukebox/components/rpc_command_alias.py

```diff
@@ -57,6 +63,18 @@
         'plugin': 'ctrl',
         'method': 'toggle',
         'ignore_card_removal_action': True},
+    'shuffle': {
+        'package': 'player',
+        'plugin': 'ctrl',
+        'method': 'shuffle',
+        'note': 'Toggles Shuffle',
+        'ignore_card_removal_action': True},
+    'repeat': {
+        'package': 'player',
+        'plugin': 'ctrl',
+        'method': 'repeat',
```

Comment:
Should this be `repeatmode` with parameter "repeat" or "single"?

Reply:
Yes. I will address this with another commit